core/src/utils/forms/compare-with-utils.ts

```diff
@@ -0,0 +1,44 @@
+type CompareFn = (currentValue: any, compareValue: any) => boolean;
+
+/**
+ * Uses the compareWith param to compare two values to determine if they are equal.
+ *
+ * @param currentValue A current value of the control.
```

Comment:
```suggestion
 * @param currentValue The current value of the control.
```

Nit: We say "The current" for the function below